From: Kefeng Wang <wangkefeng.wang@huawei.com>
Date: Mon, 26 Sep 2016 15:36:50 +0800
Subject: arm64: Kconfig: select OF/ACPI_NUMA under NUMA config
Git-commit: 0c2a6cce1776ed041b71a305edcb7d89923743e1
Patch-mainline: v4.9-rc1
References: fate#322061

Move OF_NUMA select under NUMA config, and select ACPI_NUMA
when ACPI enabled.

Signed-off-by: Kefeng Wang <wangkefeng.wang@huawei.com>
Signed-off-by: Will Deacon <will.deacon@arm.com>
Signed-off-by: Matthias Brugger <mbrugger@suse.com>
---
 arch/arm64/Kconfig |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/arch/arm64/Kconfig
+++ b/arch/arm64/Kconfig
@@ -97,7 +97,6 @@ config ARM64
 	select SYSCTL_EXCEPTION_TRACE
 	select HAVE_CONTEXT_TRACKING
 	select HAVE_ARM_SMCCC
-	select OF_NUMA if NUMA && OF
 	help
 	  ARM 64-bit (AArch64) Linux support.
 
@@ -541,6 +540,8 @@ config HOTPLUG_CPU
 config NUMA
 	bool "Numa Memory Allocation and Scheduler Support"
 	depends on SMP
+	select ACPI_NUMA if ACPI
+	select OF_NUMA
 	help
 	  Enable NUMA (Non Uniform Memory Access) support.
 
